src/IO.cpp

```diff
@@ -42,45 +42,44 @@ namespace IO {
     + Write agents to file - see AgentContainer::WritePlotFile().
 */
 void writePlotFile (const AgentContainer& pc,                      /*!< Agent (particle) container */
-                    const CensusData& censusData,                  /*!< Contains census data */
+                    const iMultiFab* unit_mf_ptr,                  /*!< MultiFabs to write out */
+                    const iMultiFab* FIPS_mf_ptr,                  /*!< MultiFabs to write out */
+                    const iMultiFab* comm_mf_ptr,                  /*!< MultiFabs to write out */
                     const int num_diseases,                        /*!< Number of diseases */
                     const std::vector<std::string>& disease_names, /*!< Names of diseases */
                     const Real cur_time,                           /*!< current time */
                     const int step /*!< Current step */) {
     amrex::Print() << "Writing plotfile \n";
 
-    static const int ncomp_d = 5;
-    static const int ncomp = ncomp_d * num_diseases + 4;
+    static const Vector<std::string> status_names = {"total", "never_infected", "infected", "immune", "susceptible", "dead"};
```

Comment:
@stevenhofmeyr Why did you include `"dead"` in the `status_names`? I think `AgentContainer::generateCellData` doesn't include `dead` agents.